eng/_util/cmd/run-builder/run-builder_windows.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//go:build windows
+// +build windows
+
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+func init() {
+	enableSystemWideFIPS = func() (func(), error) {
```

Comment:
On windows it is not possible to enable FIPS mode in a per-process basis, so this function affects all processes running in the same VM. I'm wondering if this could cause issues in case there are parallel non-fips jobs running in the same VM. I'm probably overthinking, and Azure spins a new VM for every job, but asking just in case.

Reply:
Yeah, each AzDO build always has its own VM, even when we are also running in a container. We eventually got to this point for security and cross-repo-interference reasons. There is a higher throughput system the .NET Engineering team has that I believe shares machines more readily, and there I think we'd need to be more careful, but we aren't using it. ([Helix](https://github.com/dotnet/arcade/blob/main/Documentation/Helix.md))